cc/concurrent_selfplay.cc

```diff
@@ -906,25 +936,25 @@ void OutputThread::Run() {
 
 void OutputThread::WriteOutputs(int game_id,
                                 std::unique_ptr<SelfplayGame> selfplay_game) {
-  auto output_name = GetOutputName(game_id);
   auto now = absl::Now();
+  auto output_name = GetOutputName(game_id);
   auto* game = selfplay_game->game();
-  game->AddComment(absl::StrCat(
-      "Inferences: [", absl::StrJoin(selfplay_game->models_used(), ", "), "]"));
```

Comment:
done. I stopped looking at the SGFs after I added `inspect_example.py` since it can displays pretty much everything I'm interested in directly from the training examples